src/main/io/rc_controls.c

```diff
@@ -342,7 +343,16 @@ void updateActivatedModes(modeActivationCondition_t *modeActivationConditions)
         modeActivationCondition_t *modeActivationCondition = &modeActivationConditions[index];
 
         if (isRangeActive(modeActivationCondition->auxChannelIndex, &modeActivationCondition->range)) {
```

Comment:
The definition of 'stable' is important.
If the signal is used for switching, it is more important how long is the value stable, not how stable is it.
It may be better to choose 'acceptable' channel noise and detect how long the signal is constant (within this noise interval).
Channel value may be taken as average of values that are stable (within min/max) and number of samples within min/max taken as 'stability'. And also ignore isolated outliers to improve spike noise imunity.

But that is probably unnecessarily complicated..